ci-operator/jobs/redhat-operator-ecosystem/playground/redhat-operator-ecosystem-playground-cvp-ocp-4.4-periodics.yaml

```diff
@@ -72,3 +75,6 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+    - name: cluster-secrets-aws-cpaas
+      secret:
+        secretName: cluster-secrets-aws-cpaas
```

Comment:
Same here, this is specified via the` cluster_profile` field in the test item in ci-op config, Instead of `aws` that's there now you'd use `aws-cpaas` (enabled by https://github.com/openshift/ci-tools/pull/1548) and regenerate the jobs)